build.gradle

```diff
@@ -81,10 +95,11 @@ ext {
   guiceVer = "3.0"
   jacksonVer = "2.6.3"
   junitVer = "4.11"
-  mesosVer = "1.0.0"
+  mesosVer = "1.1.0-SNAPSHOT"
```

Comment:
master is now on 0.28.0.  We still depend on libmesos.so on the host.  So if you try to deploy this it won't work on anything except 1.8.  Is this the intended policy?  If so we will need to address this by packaging libraries before merging this.